build-info-extractor/src/main/java/org/jfrog/build/extractor/clientConfiguration/ClientConfigurationFields.java

```diff
@@ -8,6 +8,8 @@ public interface ClientConfigurationFields {
     String USERNAME = "username";
     String HOST = "host";
     String PORT = "port";
+    String HTTPS = "https";
+    String NO_PROXY_DOMAIN = "noProxyDomain";
```

Comment:
This should be a comma-separated list of hosts, not a single domain

Reply:
Removed in favor of system property